mass_mailing_custom_unsubscribe/README.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Customizable unsubscribe link on mass mailing emails
+====================================================
+
+With this module you can set a custom unsubscribe link append at bottom of mass
+mailing emails.
+
+
+Installation
+============
+
+Nothing special to do
+
+
+Configuration
+=============
+
+To configure unsubscribe label go to Setting > Technical > Parameters > System parameters
+and add a 'mass_mailing.unsubscribe.label' parameter with html to set at bottom
+of mass emailing emails. Including '%(url)s' variable where unsubscribe link
+
+For example:
+```
+<small>You can unsubscribe <a href="%(url)s">here</a></small>
+
+```
+
+Additionally, you can disable this link if you set this parameter to 'False'
+
+
+Usage
+=====
+
+Nothing special to do
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/205/8.0
+
+
+Known issues / Roadmap
+======================
+
+* This custom html is not translatable, so you can define the same text in
```

Comment:
I've change the text to be more understandable. I mean that, as this text is not translatable, admin can define several times, one language per line. I've add an example for clarity